### ISSUE TITLE
Missing dependency of gdal2 on json-c

### ISSUE THREAD

**Issue by bentrm** [OSGeo/homebrew-osgeo4mac#340]
I noticed this configuring a new machine and installing gdal2 without any options like:

````
$ brew install osgeo/osgeo4mac/gdal2
````

And after completion just to test out the binaries are actually there (and creating a symbolic link to the correct version of the proj4 libs):
````
$ ogr2ogr
dyld: Library not loaded: /usr/local/opt/json-c/lib/libjson-c.2.dylib
  Referenced from: /usr/local/opt/gdal2/bin/ogr2ogr
  Reason: image not found
Abort trap: 6
````

It looks like `json-c` is only installed with option `--with-complete` but is required in the basic setup as well?

Just doing `$ brew install json-c` fixed this.

**Comment by nickrobison** [comment]
Hi there, thanks for reporting!

I believe #338 should resolve the issue you're experiencing. Hopefully it'll be merged in a couple of days.

Until then, glad you were able to come up with a workaround.

**Comment by 3nids** [comment]
fixed in d9dedee 
kudos to @nickrobison
